src/jobs/consumers/notifications/smtp-notifications.job.consumer.ts

```diff
@@ -4,24 +4,36 @@ import { InjectRepository } from '@nestjs/typeorm';
 import { Repository } from 'typeorm';
 import { Notification } from 'src/models/notifications/entities/notification.entity';
 import { DeliveryStatus } from 'src/common/constants/notifications';
+import { SmtpService } from 'src/services/email/smtp.service';
 
 @Processor('smtpNotifications')
 export class SmtpNotificationConsumer {
   constructor(
     @InjectRepository(Notification)
     private readonly notificationRepository: Repository<Notification>,
+    private readonly smtpService: SmtpService,
   ) {}
 
   @Process()
   async processSmtpNotificationQueue(job: Job): Promise<void> {
-    const notification = job.data;
+    const id = job.data;
+    const notification = (await this.getNotificationById(id))[0];
     try {
       // TODO: Send the email, update notification values
```

Comment:
Remove TODO

Reply:
This has been removed in the latest commit